File: Python/Publicacion.py
class Publicacion():
    _titulo = ""
    _precio = 0.0

    def __init__(self, titulo, precio):
        self.titulo = titulo
        self.precio = precio

    def getTitulo(self):
        return self.titulo

    def getPrecio(self):
        return self.precio

    def setTitulo(self, titulo):
        self.titulo = titulo

    def setPrecio(self, precio):
        self.precio = precio

    def __str__(self):
        return "Estado \n" + "Titulo: " + self.titulo + " Precio: " + str(self.precio)

File: Java/Publicacion.java
public class Publicacion {

    protected String titulo;
    protected float precio;

    public Publicacion(String titulo, float precio) {
        this.titulo = titulo;
        this.precio = precio;
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public float getPrecio() {
        return precio;
    }

    public void setPrecio(float precio) {
        this.precio = precio;
    }

    public String toString() {
        return "Estado: \n" + "Titulo: " + titulo + " Precio: " + precio;
    }
}